akka-docs/rst/java/code/docs/persistence/LambdaPersistenceDocTest.java

```diff
@@ -138,10 +129,10 @@ public void preStart() throws Exception {
 
   static Object fullyDisabledRecoveyExample = new Object() {
       abstract class MyPersistentActor1 extends UntypedPersistentActor {
-          //#recover-fully-disabled
+          //#recover-disabled
```

Comment:
note that you already have a `#recovery-disabled` above in this file, please verify that the generated docs looks good